labelImg.py

```diff
@@ -1300,10 +1306,13 @@ def change_save_dir_dialog(self, _value=False):
         if dir_path is not None and len(dir_path) > 1:
             self.default_save_dir = dir_path
 
+        self.show_bounding_box_from_annotation_file(self.file_path)
```

Comment:
why need this?

Reply:
if there are existing annotations in the new directory, it shows the bbox annotations for the current file ?